### PR TITLE
UI improvements

### PR DESCRIPTION
**Pull request by jmartinm**

### CODE REVIEW

**Comment by jmartinm** [comment]
Pending styling the breadcrums.

**Comment by jmartinm** [comment]
This should be now ready.

File: inspirehep/modules/records/wrappers.py [hunk 1]
```diff
@@ -22,16 +22,47 @@
...
 from inspirehep.modules.search import JobsSearch
 
 
-class JobsRecord(Record):
+class LiteratureRecord(ESRecord):
+    """Record class specialized for literature records."""
+
+    @property
+    def title(self):
+        """."""
```
Comment (jacquerie) [inline inspirehep/modules/records/wrappers.py]: Missing docstring.

File: inspirehep/modules/theme/jinja2filters.py [hunk 2]
```diff
@@ -33,7 +33,9 @@
 
 import six
 from flask import current_app
+from flask import url_for
```
Comment (jacquerie) [inline inspirehep/modules/theme/jinja2filters.py]: Can be merged with the guy above.

File: inspirehep/modules/theme/templates/inspirehep_theme/format/record/authors/Author_HTML_detailed.html [hunk 3]
```diff
@@ -24,6 +24,9 @@
 
 {% extends config.SEARCH_UI_BASE_TEMPLATE %}
 
+{% set title=record.title %}
+
+
```
Comment (jacquerie) [inline inspirehep/modules/theme/templates/inspirehep_theme/format/record/authors/Author_HTML_detailed.html]: Extra space.

File: inspirehep/modules/theme/static/scss/format/detailed-record.scss [hunk 4]
```diff
@@ -504,3 +504,14 @@ h1.record-detailed-title, .record-header label, #record-journal,
     }
 }
 
+.detailed-record-breadcrum {
```
Comment (jacquerie) [inline inspirehep/modules/theme/static/scss/format/detailed-record.scss]: s/breadcrum/breadcrumb/

File: inspirehep/modules/theme/static/scss/format/detailed-record.scss [hunk 5]
```diff
@@ -504,3 +504,14 @@ h1.record-detailed-title, .record-header label, #record-journal,
...
+    font-size: 13px;
+    margin-bottom: 0px;
+
+    .fa {
+        font-size: 10px;
+        color: $link-color;
+    }
+
+}
+
```
Comment (jacquerie) [inline inspirehep/modules/theme/static/scss/format/detailed-record.scss]: Some weird spacing here.

**Pull request merged**